elytron/src/main/java/org/wildfly/extension/elytron/DirContextDefinition.java

```diff
@@ -126,6 +126,7 @@ class DirContextDefinition extends SimpleResourceDefinition {
 
     static final SimpleAttributeDefinition MODULE = new SimpleAttributeDefinitionBuilder(ElytronDescriptionConstants.MODULE, ModelType.STRING, true)
             .setAllowExpression(true)
+            .setCorrector(ModuleIdentifierUtil.MODULE_NAME_CORRECTOR)
```

Comment:
We can safely now remove the request to canononicalize the module at: 
https://github.com/wildfly/wildfly-core/blob/481cbde8dc6ac6a2a8b29353f6b85879db54a09d/elytron/src/main/java/org/wildfly/extension/elytron/DirContextDefinition.java#L163

This is a private method, and it will use the value of the module attribute already canonicalized